modules/decision_reviews/app/controllers/decision_reviews/v1/supplemental_claims_controller.rb

```diff
@@ -91,7 +91,10 @@ def handle_personal_info_error(e)
 
       def process_submission
         req_body_obj = request_body_hash.is_a?(String) ? JSON.parse(request_body_hash) : request_body_hash
-        saved_claim_request_body = req_body_obj.to_json # serialize before request body is modified
+        # For now, we have to address schema issues before serializing, since our SavedClaim model
+        # uses a copy of the same Lighthouse schema to validate the data before saving.
+        req_body_obj = normalize_evidence_retrieval_for_lighthouse_schema(req_body_obj)
+        saved_claim_request_body = req_body_obj.to_json
```

Comment:
This method is getting long 😅. Not a blocker, but a refactor to make this method less procedural and break it out into smaller methods might be good.